integration-test-groups/foundation/controlbus/src/test/java/org/apache/camel/quarkus/component/controlbus/it/ControlbusTest.java

```diff
@@ -19,13 +19,22 @@
 import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 @QuarkusTest
 class ControlbusTest {
 
+    @BeforeEach
+    public void startRoute() {
+        String status = RestAssured.get("/controlbus/status").asString();
+        if ("Stopped".equals(status)) {
+            RestAssured.get("/controlbus/start");
+        }
+    }
+
     @Test
-    public void test() {
+    public void testStopStart() {
         RestAssured.given()
                 .contentType(ContentType.TEXT).get("/controlbus/status")
                 .then().body(org.hamcrest.CoreMatchers.equalTo("Started"));
```

Comment:
Nitpick - You can add a static import for `equalTo`. Makes things a bit easier to read without the package prefix.

Reply:
Good point. I refactored that test and the jvm test.
https://github.com/apache/camel-quarkus/pull/4200/files#diff-c93d82c8522c8189a7514a631276500d49ef7869fec288fbfb44ba92caf2231fR25